src/main/java/appeng/me/cache/SpatialPylonCache.java

```diff
@@ -41,15 +41,15 @@
 public class SpatialPylonCache implements ISpatialCache
 {
 
-	final IGrid myGrid;
-	long powerRequired = 0;
-	double efficiency = 0.0;
-	DimensionalCoord captureMin;
-	DimensionalCoord captureMax;
-	boolean isValid = false;
-	List<TileSpatialIOPort> ioPorts = new LinkedList<TileSpatialIOPort>();
-	HashMap<SpatialPylonCluster, SpatialPylonCluster> clusters = new HashMap<SpatialPylonCluster, SpatialPylonCluster>();
-	boolean needsUpdate = false;
+	private final IGrid myGrid;
+	private long powerRequired = 0;
+	private double efficiency = 0.0;
+	private DimensionalCoord captureMin;
+	private DimensionalCoord captureMax;
+	private boolean isValid = false;
+	private List<TileSpatialIOPort> ioPorts = new LinkedList<TileSpatialIOPort>();
+	private HashMap<SpatialPylonCluster, SpatialPylonCluster> clusters = new HashMap<SpatialPylonCluster, SpatialPylonCluster>();
+	private final boolean needsUpdate = false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "needsUpdate" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)